src/librustc_codegen_ssa/mir/block.rs

```diff
@@ -651,6 +651,12 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
         }
 
         if intrinsic.is_some() && intrinsic != Some("drop_in_place") {
+            let intrinsic = intrinsic.unwrap();
+
+            if !bx.is_codegen_intrinsic(intrinsic, &args, self.instance) {
```

Comment:
Maybe add a short comment explaining what this is checking for?

Reply:
Done